pvsystemprofiler/latitude_study.py

```diff
@@ -25,6 +24,8 @@ def __init__(self, data_handler, daytime_threshold=[0.001, 0.001, 0.001, 0.001],
 
         self.data_handler = data_handler
         self.daytime_threshold = daytime_threshold
```

Comment:
Same comment here as on the previous commit. Please use `is None` construction and avoid the extra variable assignment.